<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>811</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>481</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridParameters">
      <item row="3" column="0">
       <widget class="QLabel" name="lblHistory">
        <property name="text">
         <string>History:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="hlInput">
        <item>
         <widget class="QLabel" name="lblFirstParameter">
          <property name="text">
           <string>Volume:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ln_edFirstParameter"/>
        </item>
        <item>
         <widget class="QLabel" name="lblSecondParameter">
          <property name="text">
           <string>Temperature:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ln_edSecondParameter"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblSetParameters">
        <property name="text">
         <string>Set parameters:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="hlCalculate">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCalculate">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QTableView" name="tbl_vwHistory"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridGasComponents">
      <item row="0" column="0">
       <widget class="QLabel" name="lblComponents">
        <property name="text">
         <string>Components:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tbl_vwComponents">
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="hlComponent">
        <item>
         <widget class="QComboBox" name="cmbAddComponent">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ln_edComponentPart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="hlAddComponent">
        <item>
         <widget class="QPushButton" name="btnAddComponent">
          <property name="text">
           <string>Add component</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveComponent">
          <property name="text">
           <string>Remove component</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="vertSeparator">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>20</width>
       <height>461</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>&amp;Configuration</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
